hub/core/compression/jpeg.py

```diff
@@ -53,15 +54,15 @@ def encode_single_image(self, image: np.ndarray) -> bytes:
             )
             return buffer.getvalue()
 
-    def decode_single_image(self, buf: bytes) -> np.ndarray:
+    def decode_single_image(self, buf: Union[int, memoryview, bytes]) -> np.ndarray:
         """
         Decode single image from buffer.
 
         Example:
             imgs_decoded = jpeg_codec.decode(imgs_encoded)
 
         Args:
-            buf (bytes): Encoded image
+            buf (Union[int, memoryview, bytes]): Encoded image
```

Comment:
```suggestion
            buf (int, memoryview, bytes): Encoded image
```

can you make these all consistent? 

run `darglint .` to see if these are all acceptable. if they are all acceptable you should do the above style, otherwise just follow `darglint`'s